src/Infrastructure/Service/ProjectInfo/ComposerProjectInfoFetcher.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\ProjectInfo;
+
+use SprykerSdk\Sdk\Core\Appplication\Dependency\Repository\SettingRepositoryInterface;
+use SprykerSdk\Sdk\Core\Appplication\Dependency\Service\ProjectInfo\ProjectInfoInterface;
+
+class ComposerProjectInfoFetcher implements ProjectInfoFetcherStrategyInterface
+{
+    /**
+     * @var string
+     */
+    protected const PROJECT_DIR_KEY = 'project_dir';
```

Comment:
Use `SettingPath::PROJECT_DIR` instead. The same for the rest of existing code that are use `project_dir` and `report_dir` setting keys